src/utils/carbonApi.ts

```diff
@@ -29,26 +30,15 @@ export type RoiRow = {
   id: string;
 };
 
-let BASE_URL = '/api/';
-
-if (import.meta.env.VITE_DEV_MODE) {
-  BASE_URL = 'https://app.carbondefi.xyz/api/';
-}
-
-const carbonApiAxios = axios.create({
-  baseURL: BASE_URL,
-});
-
 const newApiAxios = axios.create({
-  baseURL: 'https://api.carbondefi.xyz/v1/',
+  baseURL: config.carbonApi,
 });
 
 const carbonApi = {
   getCheck: async (): Promise<boolean> => {
-    if (import.meta.env.VITE_DEV_MODE) {
-      return false;
-    }
-    const { data } = await carbonApiAxios.get<boolean>('/check');
+    if (config.mode === 'development') return false;
+    const baseURL = config.appUrl + '/api/';
+    const { data } = await axios.create({ baseURL }).get<boolean>('/check');
```

Comment:
Just thinking out loud. We could use a new axios instance but with relative path? Then it would work on previews with preview CF functions.

Reply:
I'm not familiar with cloudflare lambda. They are deployed alongside the preview link ?
I think it's much better to use relative path as you mentioned because it's part of the repo.